app/controllers/users_controller.rb

```diff
@@ -56,6 +56,7 @@ def user_params
 
   def load_user
     @user = User.find_by id: params[:id]
+    @user.update_attribute(:last_sign_in_at, Time.zone.now)
```

Comment:
Another writing style 
[@user.update_attribute last_sign_in_at: Time.zone.now] 
will cause error when calling it.

Reply:
@chan227 when use update_attributes, it will @user.update_attributes last_sign_in_at: Time.zone.now
=> it will run callback and validate when you update
when use update_attribute, it will like above @user.update_attribute(:last_sign_in_at, Time.zone.now)
-> it still run callback but not run validate

Reply:
you should update     @user.update_attribute(:last_sign_in_at, Time.zone.now) under login action. not here
For example
if ...
   login @user 
   @user.update_attribute(:last_sign_in_at, Time.zone.now)

Reply:
a bit confusing...not sure how to do
Is this 'def load user' counted as login class?
This coding can only work on this class, other class can work as well but the time will not be updated.

Reply:
it should update below line 8
https://github.com/awesome-academy/malay-group-1/blob/master/app/controllers/sessions_controller.rb#L8

Reply:
![image](https://user-images.githubusercontent.com/68467813/93746046-0c639b80-fc27-11ea-8802-a7f378efeb62.png)
Errors occured...
![image](https://user-images.githubusercontent.com/68467813/93746200-50ef3700-fc27-11ea-9b7d-41c85bb8c9db.png)

Reply:
user not @user

Reply:
done